ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/WearApp.kt

```diff
@@ -228,30 +227,40 @@ fun WearApp(
 
                     val focusRequester = remember { FocusRequester() }
 
-                    val context = LocalContext.current
+                    val menuItemNameToClickHandlerMap = sortedMapOf(
```

Comment:
sortedMap seems like an unlikely realistic choice.  What about LinkedHashMap, so insertion order is respected?